iFixit/react-commerce
# Troubleshooting: Add nav tabs

Both these URLs may be missing if there's not a sensible place to link to, so this will disable the tab if there's no URL for it.

## QA Notes
Please confirm that the tabs look reasonable and link to the appropriate places.

Example URL which should have both a Parts and a Guide link:
https://react-commerce-git-vulcan-add-area-links-ifixit.vercel.app/Vulcan/iPhone_Won't_Turn_On
Example URL which should have a working Guide link and a disabled Parts link:
https://react-commerce-git-vulcan-add-area-links-ifixit.vercel.app/Vulcan/Washing_Machine_Not_Spinning
Example URL which should have both a disabled Guide link and a disabled Parts link:
https://react-commerce-git-vulcan-add-area-links-ifixit.vercel.app/Vulcan/troubleshooting-test

Closes iFixit/ifixit#46652
